scheduler/src/main/java/org/apache/falcon/execution/SchedulerUtil.java

```diff
@@ -27,6 +28,14 @@
 
     private static final long MINUTE_IN_MS = 60 * 1000L;
     private static final long HOUR_IN_MS = 60 * MINUTE_IN_MS;
+    public static final String MINUTELY_PROCESS_FREQUENCY_POLLING_IN_MILLIS =
+            "falcon.scheduler.minutely.process.polling.frequency.millis";
```

Comment:
This is just for initial go, later we can discuss and see what all we can add and discuss how it will go.